src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -545,6 +545,8 @@ private function getArgumentsAsPhp(\DOMElement $node, $name, $file, $lowercase =
                     }
                     break;
                 case 'tagged':
+                case 'tagged_iterator':
+                    @trigger_error(sprintf('Type "%s" of Tag <%s> in "%s" is deprecated since Symfony 4.4, use "tagged_iterator" instead.', $arg->getAttribute('type'), $name, $file), E_USER_DEPRECATED);
```

Comment:
one case up

Reply:
I'm afraid I don't understand. Because if we do this:
```
...
case 'tagged':
    @trigger_error(sprintf('Type "%s" of Tag <%s> in "%s" is deprecated since Symfony 4.4, use "tagged_iterator" instead.', $arg->getAttribute('type'), $name, $file), E_USER_DEPRECATED);
    break;
...
```

the `case 'tagged_locator':` wouldn't be reached, right?

Reply:
see https://3v4l.org/psY8S :)

Reply:
Yes, leaving out the `break;` would help, but then the fabbot.io check will complain about it.

Reply:
ok, adding a `// no break` will make fabbot.io happy :-)

Reply:
Thanks @ro0NL for your help :-)